test/src/test/kotlin/graphql/nadel/tests/CentralSchemaTesting.kt

```diff
@@ -36,7 +36,7 @@ val File.parents: Sequence<File>
  * Set this to true to run the query. You'll have to modify the [ServiceExecutionFactory] to
  * actually return something e.g. response from Splunk etc.
  */
-const val runQuery = false
+const val runQuery = true
```

Comment:
hey, is this deliberate? the doco above says the default is `false

Reply:
Yup mistake, I'll fix revert it.